core/api/src/services/oathkeeper/index.ts

```
@@ -9,18 +9,24 @@ import {
   UnknownOathkeeperServiceError,
 } from "@/domain/oathkeeper/errors"
 
-export const sendOathkeeperRequestGraphql = async (
-  token: AuthToken | undefined,
-): Promise<JwtToken | OathkeeperError> => {
+export const sendOathkeeperRequestGraphql = async ({
+  bearerToken,
+  apiKey,
+}: {
+  bearerToken: AuthToken | undefined
+  apiKey: AuthToken | undefined
```

should we return an error if both are set?